packages/uswds/_index.scss

```diff
@@ -47,6 +47,7 @@
 @forward "usa-tag/src/styles";
 @forward "usa-tooltip/src/styles";
 @forward "uswds-form-controls";
+@forward "usa-input-mask/src/styles";
```

Comment:
I am thinking this should be included in uswds-form-controls instead. Can you move this `@forward` reference to packages/uswds-form-controls/_index.scss?

Reply:
Done